sonar-project.properties

```diff
@@ -1 +1,51 @@
 sonar.projectKey=icanbwell_fhir-server
+sonar.sources=src
+sonar.python.version=3.8
+sonar.exclusions=\
+  **/env.template,\
+  **/*.jinja2,\
+  **/*.xml,\
+  **/*.zip,\
+  **/*.info,\
+  **/*.sch,\
+  **/*.xsd,\
+  **/*.json,\
+  **/*.html,\
+  **/*.csv,\
+  **/*.css,\
+  **/*.d.ts,\
+  src/fhir/classes/**,\
+  src/fhir/generator/search_parameters.py,\
+  src/graphql/v1/generator/search_parameters.py,\
+  src/graphql/v1/resolvers/backbone_elements/**,\
+  src/graphql/v1/resolvers/complex_types/**,\
+  src/graphql/v1/resolvers/extensions/**,\
+  src/graphql/v1/resolvers/resources/**,\
+  src/graphql/v1/schemas/**,\
+  src/graphql/v2/resolvers/backbone_elements/**,\
+  src/graphql/v2/resolvers/complex_types/**,\
+  src/graphql/v2/resolvers/extensions/**,\
+  src/graphql/v2/resolvers/resources/**,\
+  src/graphql/v2/schemas/**,\
+  src/images/**,\
+  src/searchParameters/searchParameters.js,\
+  src/views/**
+sonar.coverage.exclusions=\
+  **/env.template,\
+  **/*.jinja2,\
+  **/*.xml,\
+  **/*.zip,\
+  **/*.info,\
+  **/*.sch,\
+  **/*.xsd,\
+  **/*.json,\
+  **/*.html,\
+  **/*.csv,\
+  **/*.css,\
+  **/*.d.ts,\
+  **/*.py,\
+  src/app.test.js,\
+  src/dist/**,\
+  src/tests/**,\
```

Comment:
I think this should be extracted into separate sonar property. https://docs.sonarqube.org/9.6/project-administration/narrowing-the-focus/#:~:text=Why%20is%20test%20code%20scoped%20separately%3F
`sonar.tests=src/tests/**`

Reply:
Updated